pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -814,3 +847,80 @@ def plot_channel_contributions_grid(
             ylabel="contribution",
         )
         return fig
+
+    def _validate_data(self, X, y=None):
+        return X
+
+    def sample_posterior_predictive(
+        self,
+        X_pred,
+        extend_idata,
+        combined,
+        include_last_observations: bool = False,
+        original_scale: bool = True,
+        **kwargs,
```

Comment:
Maybe call this `sample_posterior_predictive_kwargs` (or something more informative?) To indicate where these additional parameters will be used.

Reply:
Or as below a better description: "Additional arguments to pass to sample_posterior_predictive method"

Reply:
done. This isn't changed for the original model_builder methods however. Would you want that to change in this PR?

Reply:
Ok I think we can leave it as it is in the docstrings for now. Let's wait for feedback!